pkg/config/config.go

```diff
@@ -270,6 +271,10 @@ func (c *Config) Connection() (connection *sdk.Connection, err error) {
 	}
 	builder.Insecure(c.Insecure)
 
+	if c.RhRegion != "" {
+		builder.RhRegion(c.RhRegion)
```

Comment:
kill it. We connect to the correct region because cfg.URL is pointed to the correct region url. The sdk ConnectionBuilder will not be aware of the concept of a region.

Reply:
Removed